docs/design/integrate-with-cc-tools/README.md

```diff
@@ -0,0 +1,308 @@
+# FPC and CC-tools integration
+
+The goal of this project is to combine the strength of [cc-tools](https://github.com/hyperledger-labs/cc-tools) with the enhanced security capabilities of [Fabric Private Chaincode (FPC)](https://github.com/hyperledger/fabric-private-chaincode).
+We describe the integration design, including the user-facing changes, the required modifications to the cc-tools and FPC, and limitations.
+Finally, we present a guide that shows how to build, deploy, and run a Fabric chaincode built using cc-tools and FPC.
```

Comment:
Can you check this paragraph, which is a teaser what to expect in this document, that we present everything in the right order. Limitations and Future work are the last sections, not the guide (user experience).

Reply:
Done